File: p10_u3.cs
using System;

namespace MyApp // p10_u3
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int ht, hex;
            double pago, sueldosem = 0;
            Console.WriteLine("¿Cuantas horas trabajaste?");
            ht = Convert.ToInt32(Convert.ToDouble(Console.ReadLine()));
            Console.WriteLine("¿Cuanto es el pago por hora?");
            pago = Convert.ToInt32(Convert.ToInt32(Console.ReadLine()));

            if (ht <= 40)
            {
                sueldosem = ht * pago;
                Console.WriteLine("Recibiras un total de $" + sueldosem);
            }
            else
            {
                hex = ht - 40;
                sueldosem = 40 * pago + hex * (pago * 2);
                Console.WriteLine("Por trabajar " + hex + " horas extra, usted recibe un total de $" + sueldosem);


            }
        }
    }
}

File: P10_u3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p10_u3;
import java.util.Scanner;
/**
 *
 * @author richy
 */
public class P10_u3 {

    public static void main(String[] args) {
        
            Scanner sc= new Scanner(System.in);
            int ht, hex;
            double pago, sueldosem = 0;
            System.out.println("¿Cuantas horas trabajaste?");
            ht = sc.nextInt();
            System.out.println("¿Cuanto es el pago por hora?");
            pago = sc.nextInt();

            if (ht <= 40 )
            {
                sueldosem = ht * pago;
                System.out.println("Recibiras un total de $" + sueldosem);
            }
            else
            {
                hex = ht - 40;
                sueldosem = 40 * pago + hex * (pago * 2);
                System.out.println("Por trabajar " + hex + " horas extra, usted recibe un total de $" + sueldosem);


            }
    }
}
